rpm/nemo-qml-plugin-contextkit-qt5.spec

```diff
@@ -2,8 +2,8 @@ Summary: ContextKit QML bindings
 Name: nemo-qml-plugin-contextkit-qt5
 Version: 2.0
 Release: 2
```

Comment:
Hmm, though, I think this should be 0 or 1. Probably doesn't matter for OBS building at all. :thinking:

Reply:
OBS replaces that with something so doesn't matter, though most commonly it's just 1 here.